src/components/pages/dashboard/dashboard.js

```diff
@@ -400,7 +402,8 @@ export class Dashboard extends Component {
                   mapKeyIsPending={azureMapsKeyIsPending}
                   isPending={devicesIsPending || analyticsIsPending}
                   error={azureMapsKeyError || devicesError || analyticsError}
-                  t={t} />
+                  t={t}
+                  logEvent={logEvent} />
```

Comment:
What is the reason passing logEvent from dashboard to each panel instead of each panel itself have a container and dispatch logEvent?
Although logEvent does not have payload in this case, we may want to add metadata for each panel later and these metadata belong to panel's state rather than dashboard's state. 
Also, if the component is reused by other pages, other pages should also implement and pass logEvent to each components. (this is not the case for a panel because it is not shared currently, but I think having a container for a panel would be preferable pattern)

Reply:
the reason was to simplify the code because we are only using the panels from the dashboard, so it was simpler to not add extra container files. I don't have a strong preference either way, but for components that were only used once this seemed simpler

Reply:
since I'm taking out the call to log event from the dashboard code I will add containers